src/execution/IncrementalGraph.ts

```diff
@@ -49,23 +46,25 @@ export class IncrementalGraph {
     return this._promoteNonEmptyToRoot(initialResultChildren);
   }
 
-  addCompletedReconcilableDeferredGroupedFieldSet(
-    reconcilableResult: ReconcilableDeferredGroupedFieldSetResult,
+  addCompletedReconcilableExecutionGroup(
```

Comment:
should this function be renamed to `addCompletedSuccessfulExecutionGroup`?

Reply:
Yes!

db4482a6